client/src/main/java/com/vesoft/nebula/client/graph/net/NebulaPool.java

```diff
@@ -70,6 +70,7 @@ public boolean init(List<HostAddress> addresses, NebulaPoolConfig config)
         this.objectPool = new GenericObjectPool<>(objectPool);
         GenericObjectPoolConfig objConfig = new GenericObjectPoolConfig();
         objConfig.setMinIdle(config.getMinConnSize());
+        objConfig.setMaxIdle(config.getMinConnSize());
```

Comment:
Thanks for your contribution! 
The object pool instructions
`
If the configured value of minIdle is greater than the configured value for maxIdle then the value of maxIdle will be used instead.
`